helix-view/src/view.rs

```diff
@@ -215,9 +215,9 @@ impl View {
     }
 
     pub fn ensure_cursor_in_view(&mut self, doc: &Document, scrolloff: usize) {
-        if let Some((row, col)) = self.offset_coords_to_in_view(doc, scrolloff) {
+        if let Some((row, _col)) = self.offset_coords_to_in_view(doc, scrolloff) {
             self.offset.row = row;
-            self.offset.col = col;
+            self.offset.col = 0;
```

Comment:
Only included so virtual text can be tried out, will be removed before merging (altough the basic idea of avoiding horizontal view movement when softwrapping is enabled is correct)